### ISSUE TITLE
outcomes

### ISSUE THREAD

**Issue by whyzhow** [My-Mujica/LSTM-Optuna#1]
import joblib, torch, numpy as np, pandas as pd
from models.lstm_model import LSTMForecaster
from utils.data_utils import load_series_from_csv, standardize_series

# 加载 scaler 与 optuna study（示例）
scaler = joblib.load('outputs/scaler_multioutput.pkl')
study = joblib.load('outputs/optuna_study_multioutput.pkl')  

# 查看最优超参（示意）
print('best params:', study.best_trial.params)

# 构建模型（需和 best params 一致）
params = study.best_trial.params
model = LSTMForecaster(input_size=1, hidden_size=params['hidden_size'],
                       num_layers=params['num_layers'],
                       dropout=params['dropout'],
                       bidirectional=params.get('bidirectional', False),
                       horizon=14)  用你训练时的 horizon 替代
model.load_state_dict(torch.load('outputs/best_model_multioutput.pth', map_location='cpu'))
model.eval()

# 准备最后 window 做样例预测
series, df, col = load_series_from_csv('data/daily-min-temperatures.csv', value_col=None)
s_scaled, _ = standardize_series(series)
last_window = s_scaled[-params.get('seq_len', 24):]  seq_len 用 best params 或默认
x = torch.tensor(last_window[None,:,None], dtype=torch.float32)
pred_scaled = model(x).detach().numpy().ravel()
pred = scaler.inverse_transform(pred_scaled.reshape(-1,1)).ravel()
print('预测：', pred)
